pages.it/linux/deluser.md

```diff
@@ -1,7 +1,7 @@
 # deluser
 
 > Rimuovi un account utente o un utente da un gruppo.
-> Maggiori informazioni: <https://manpages.debian.org/adduser/deluser.8.html>.
+> Maggiori informazioni: <https://manpages.debian.org/latest/adduser/deluser.html>.
```

Comment:
Doesn't the `.8` specify a specific man section? The user might end up with a page from the wrong section if we remove this.

Reply:
Then how does `manned.org` do this? There aren't any sections in the link either. I assume there are only very rare cases where there are conflicting names in two sections and I think in those cases we could keep the section number.

Reply:
@marchersimon: yes the "8" is for the section.
But as some commands have got several mans from several sections, that's why the 8 is given in the default page of the debian manpage.
As far as I understood, for manned.org it is the same process. We could use the link without section when there is no confusion but can as well use the section links when necessary.
Example, for the man page of the `printf` function:

- <https://manned.org/print.1> for the shell command
- <https://manned.org/print.3> for C function

Reply:
So, since we only use links to commands, and section 8 is the default for any man pages, I think we're fine removing them everywhere.

Reply:
All right, I'm in ! ^^